### ISSUE TITLE
Check if component mounted before setting state in setTimeout

### ISSUE THREAD

**Issue by chandu0101** [japgolly/scalajs-react#66]
Hi mate ,
    How can i check if component is mounted before setting state of component in setTimeout ?, original react has this.mounted ...

```scala
dom.window.setTimeout(() => {
          if(isMounted?) {
            t.modState(_.copy(newvalue)
          }
        },3000)
```

**Comment by japgolly** [comment]
I haven't seen that in their doco. I grepped their codebase and it is called `isMounted` (not `mounted`).

This would be an easy one, could you submit a PR. It's available on components' `this` so that means the method would have to go on one of the `ComponentScope` interfaces. Should be 1 line for fix and like 3 lines to test. (Test doesn't need to worry about logic of the value, just that React does actually provide a value.)

**Comment by chandu0101** [comment]
sorry for type mate. i just sent PR ,please review it .

Thank You
Chandra

**Comment by japgolly** [comment]
Closed via 0694e5986eee3bf658d6067432889e418bf5380f
